### ISSUE TITLE
[Feature Request] Pathing issue with Summoning Bells in Housing regions between loops causes the game to lock up

### ISSUE THREAD

**Issue by jlacey-game** [ffxivcode/AutoDuty#676]
### What problem are you trying to fix?

On Preferred Summoning Bell in Between Loop Settings of the Config Tab it says "No matter what location is chosen, If there is a Summoning Bell in the location you are in when this is invoked it will go there instead" This setting is causing massive pathing issues in Housing locations. This problem has caused a huge problem with Autoretainer. I have had this problem be so bad that it crashes my game. In fact, out of the all the plugins I use... I would consider this the most painful bug because it doesn't even fail... it just keeps trying to path to a summoning that it will never reach.

### What is the solution to the problem?

Just give me the option to always have it go to the preferred location regardless of whether there is a Summoning Bell in the same location.

**Comment by erdelf** [comment]
.. so you have a location set for the summoning bell, that you don't want to retire to. And start queueing while in a housing area.. which it somehow can't reach the bell for?

**Comment by jlacey-game** [comment]
No, I have a summoning bell set. I want to go to it.  Instead of doing that, Autoduty is trying to path to another Summoning Bell in the same area and keeps failing.

**Comment by jlacey-game** [comment]
This bug is super easy to replicate. If I queue while in a house, it will exit me out of the house. The first thing it tries to do after completing a loop is if a retainer is available will be a summoning bell if a retainer action is available. Happens 100% of the time.

**Comment by erdelf** [comment]
fixed in https://github.com/ffxivcode/AutoDuty/pull/850
